app/js/components/orderEntry/OrderHeader.jsx

```diff
@@ -10,18 +10,18 @@ const OrderHeader = ({
   handleDiscontinue,
 }) => (
   <Fragment>
-    <th className="th-orderable">{orderable.toLowerCase()}</th>
-    <th>{status}</th>
+    {orderable && <th className="th-orderable">{orderable.toLowerCase()}</th>}
+    {status && <th>{status}</th>}
     <th className="order-actions-btn">
-      {order.type === 'drugorder'
+      {order.type !== 'laborder'
         && <IconButton
           iconClass="icon-pencil"
           iconTitle="EDIT"
           onClick={handleEdit}
           dataContext={order}
         />
       }
-      {order.type === 'drugorder'
+      {order.type !== 'laborder'
```

Comment:
any reason for the change of the conditional expression ??

this doesn't handle all edge cases, this applies to `line 16` too.

Reply:
ok so the edit and delete buttons were only showing for drug orders previously hence the === `drugorder` but now I also want it to show for setorder, so to get this I put up a conditional that would be run only when the order type is a set or drug order

Reply:
@Efosaok

Reply:
ok